arrow-fx-reactor/src/test/kotlin/arrow/fx/FluxKTest.kt

```diff
@@ -164,26 +173,22 @@ class FluxKTest : UnitSpec() {
 }
 
 private fun <T> FluxK.Companion.eq(): Eq<FluxKOf<T>> = object : Eq<FluxKOf<T>> {
-  override fun FluxKOf<T>.eqv(b: FluxKOf<T>): Boolean =
-    try {
-      this.value().blockFirst() == b.value().blockFirst()
-    } catch (throwable: Throwable) {
-      val errA = try {
-        this.value().blockFirst()
-        throw IllegalArgumentException()
-      } catch (err: Throwable) {
-        err
-      }
-
-      val errB = try {
-        b.value().blockFirst()
-        throw IllegalStateException()
-      } catch (err: Throwable) {
-        err
-      }
-
-      errA == errB
-    }
+  override fun FluxKOf<T>.eqv(b: FluxKOf<T>): Boolean {
+    val res1 = Try { value().timeout(Duration.ofSeconds(5)).blockFirst() }
+    val res2 = Try { b.value().timeout(Duration.ofSeconds(5)).blockFirst() }
```

Comment:
I would avoid using Try as it'll only add more "weight" to removing it later

Reply:
This is not correct anyway.. I'm going to rewrite this `Eq` instance to take into account all values within the `Flux`.